src/OpenTelemetry.Instrumentation.SqlClient/Implementation/SqlActivitySourceHelper.cs

```diff
@@ -29,6 +31,11 @@ internal class SqlActivitySourceHelper
 
         public const string MicrosoftSqlServerDatabaseSystemName = "mssql";
 
+        public static readonly IEnumerable<KeyValuePair<string, object>> CreationTags = new[]
```

Comment:
Since we are now adding the tag `db.system` on `StartActivity`, could we get remove the redundant `SetTag` at line 85 of `SqlClientDiagnosticListener.cs` and at line 141 of `SqlEventSourceListener.netfx.cs`?

Reply:
Oops, I did mean to remove that.